test/encryption/encryption.test.js

```diff
@@ -1130,49 +1118,24 @@ describe('encryption integration tests', () => {
       const [{ _id }] = await model.insertMany([{ a: 2 }]);
       const encryptedDoc = await utilClient.db('db').collection('schemas').findOne({ _id });
 
-      assert.ok(isBsonType(encryptedDoc.a, 'Binary'));
-      assert.ok(encryptedDoc.a.sub_type === 6);
+      isEncryptedValue(encryptedDoc, 'a');
 
       const doc = await model.findOne({ _id });
       assert.deepEqual(doc.a, 2);
     });
   });
 
-  describe('Encryption can be configured on the default mongoose connection', function() {
-    afterEach(async function() {
-      mongoose.deleteModel('Schema');
-      await mongoose.disconnect();
-
-    });
-    it('encrypts and decrypts', async function() {
-      const schema = new Schema({
-        a: {
-          type: Schema.Types.Int32,
-          encrypt: { keyId: [keyId], algorithm }
-
-        }
-      }, {
-        encryptionType: 'csfle'
-      });
-
-      const model = mongoose.model('Schema', schema);
-      await mongoose.connect(clusterUri, autoEncryptionOptions());
-
-      const [{ _id }] = await model.insertMany([{ a: 2 }]);
-      const encryptedDoc = await utilClient.db('db').collection('schemas').findOne({ _id });
-
-      assert.ok(isBsonType(encryptedDoc.a, 'Binary'));
-      assert.ok(encryptedDoc.a.sub_type === 6);
+  describe('Key Vault API', function() {
+    let connection;
+    let model;
 
-      const doc = await model.findOne({ _id });
-      assert.deepEqual(doc.a, 2);
+    afterEach(async function() {
+      await connection.close();
```

Comment:
```suggestion
      await connection?.close();
```

Just to be sure that this will never throw if setup somehow fails.